### PR TITLE
Fixes #13283 - Updated yum clone distributor to work with 2.8

### PR DESCRIPTION
**Pull request by parthaa**
New Distributor

### CODE REVIEW

File: pulp_katello/distributors/yum_clone_distributor/distributor.py [hunk 1]
```diff
@@ -87,9 +86,8 @@ def validate_config(self, repo, config, related_repos):
         return True, None
 
     def find_yum_distributor(self, repo_id):
-        dist_manager = RepoDistributorManager()
-        for dist in dist_manager.get_distributors(repo_id):
-            if dist["distributor_type_id"] == 'yum_distributor':
+        for dist in model.Distributor.objects(repo_id=repo_id):
+            if dist.distributor_type_id == 'yum_distributor':
```
Comment (beav) [inline pulp_katello/distributors/yum_clone_distributor/distributor.py]: maybe change this to `TYPE_ID_DISTRIBUTOR_YUM` and add import to line 24

**Comment by beav** [comment]
which RPM installs this? it will need to be updated to require 2.8 since this code won't work with older versions of Pulp.

**Comment by ehelms** [comment]
https://github.com/Katello/katello-packaging/tree/master/pulp-katello

**Comment by beav** [comment]
I tested this change using package upload to work around the unrelated sync issue. The distributor appears to fire without issue.

ACK from me

**Comment by beav** [comment]
note: when its time to cut a release, please be sure to update the dep version in the spec file to require pulp-server >= 2.8.0

**Comment by ehelms** [comment]
This should get a version bump and release in the setup.py as well?

**Comment by beav** [comment]
@ehelms it looks like tito will take care of that when bumping version, at least it appears to have from the last version bump.

**Comment by ehelms** [comment]
We don't run tito in here anymore :)

**Comment by jlsherrill** [comment]
pylint fixed here:  https://github.com/Katello/pulp-katello-plugins/pull/15

**Comment by ehelms** [comment]
This might need a rebase

**Comment by parthaa** [comment]
@ehelms rebased

**Comment by beav** [comment]
:green_apple: :green_book: :green_heart: green!

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**